src/utils/helpers.js

```diff
@@ -83,6 +83,9 @@ var helpers = {
     degree2hour: deg => {
         return deg > 0 ? deg / 15 : (deg + 360) / 15
     },
+    clamp: (val, min, max) => {
+        return val > max ? max : val < min ? min : val;
```

Comment:
This might be a bit more readable just using the builtin max/min. I.E. Math.max(Math.min(val, max), min)